webpack/ForemanInventoryUpload/ForemanInventoryConstants.js

```diff
@@ -2,4 +2,5 @@ import { translate as __ } from 'foremanReact/common/I18n';
 import { foremanUrl } from '../ForemanRhCloudHelpers';
 
 export const INVENTORY_PAGE_TITLE = __('Red Hat Inventory Uploads');
-export const INVENTORY_BASE_URL = foremanUrl('/foreman_inventory_upload/');
+export const INVENTORY_BASE_URL = () =>
+  foremanUrl('/foreman_inventory_upload/');
```

Comment:
@sharvit @mariaaga should we use functions in constants?
or this should be moved to the helper method of `inventoryUrl`?

I don't see an issue with the two approaches, though what should be the convention here? thanks

Reply:
I think it would make sense more as a helper function like `getInventoryURL` because it's not actually a constant value and you need to "get it" each time.

Reply:
agree, makes sense